.github/workflows/build-llvm.yml

```diff
@@ -82,13 +82,13 @@ jobs:
     - run: git clone --depth 1 --branch solana-rustc/13.0-2021-08-08 https://github.com/solana-labs/llvm-project.git
 #    - name: Install Ninja
 #      uses: llvm/actions/install-ninja@main
-    - run: arch -arm64 cmake -G Ninja -DLLVM_ENABLE_ASSERTIONS=On -DLLVM_ENABLE_TERMINFO=Off
```

Comment:
Why are you changing this command?

Reply:
Hyperledger arm runner is no longer running in intel emulation mode.

See https://discord.com/channels/905194001349627914/978605370484359209/978647353915473960